### PR TITLE
translate about page in italian

### PR DESCRIPTION
**Pull request by aymen94**
translated 'about' page in Italian

### CODE REVIEW

**Comment by ghost** [comment]
@aymen94：Maybe you can directly change at #1762.

@nodejs/nodejs-it

**Comment by aymen94** [comment]
[nodejs-it](https://github.com/nodejs/nodejs-it)  it is not updated since 2015
Where can I do PR?

**Comment by aymen94** [comment]
What should I do?

**Comment by ghost** [comment]
Well……Since I notice that you have a conflict translation (both of you have translated the same article), so you have to wait until https://github.com/nodejs/nodejs.org/pull/1762 is submitted, you can update your codes to the lastest and then continue your changes if possible.

**Comment by fhemberger** [comment]
@aymen94 Hi, we've merged the Italian translation. Please compare the existing version with your changes.

**Comment by aymen94** [comment]
@fhemberger  @Maledong  i fixed the page  `/it/about/resources/` and translated.
let me know 😃

**Comment by Maledong** [comment]
@aymen94: Thanks for your contribution! And please wait for the approval of@nodejs/nodejs-it 's review.

**Comment by aymen94** [comment]
@fhemberger @Maledong 
Can I make Merge?

**Comment by fhemberger** [comment]
@aymen94 We always let other members of a locale review the translation, please don't merge your own PRs.

/cc @nodejs/nodejs-it Please take a look

**Review by xela92 — COMMENTED** [review]
Hi @aymen94,
I had a quick look to your translations.

just some quick hint: 

> HTTP ha un posto di rilievo in Nodo

it should be "Node" I guess

> Solo perché il nodo è progettato senza thread

it should be "Solo perché Node è progettato senza thread"

>Le linee guida per la visualizzazione del marchio Node.js sono descritte
 nell'[Visual Guidelines]

nell'[Visual Guidelines] should be nelle [Visual Guidelines]

**Comment by aymen94** [comment]
@xela92 Thanks for the review,  Now I fix.

**Review by ghost — APPROVED** [review]
LGTM, It seems errors have all been fixed, so @xela92：Any other ideas?

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: Nit: space after dot.

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: Please replace all occurrences of Node with Node.js. It's the official way to refer to it.

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: Nit: I / O -> I/O

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: I prefer the original translation here "Dal momento che nulla si blocca, è molto facile sviluppare sistemi scalabili in Node.js"

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: Typo "Ru". The original translation sounds better. "da sistemi come la [Event Machine][] di Ruby o [Twisted][] di Python"

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: Remove "Node prende un po' dal modello a eventi."

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: "Node.js usa un [event loop][]"

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: `EventMachine :: run ()` -> `EventMachine::run ()`

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: I'd remove "nostra". "la API ..."

File: locale/it/about/resources.md
Comment (lpinca) [inline locale/it/about/resources.md]: "di rivedere" -> "di leggere".
"sull'uso consenso di Node.js" -> "su come utilizzare i loghi e il marchio Node.js".

File: locale/it/about/resources.md
Comment (lpinca) [inline locale/it/about/resources.md]: "Le linee guida visive sono descritte nelle ..."

**Comment by aymen94** [comment]
@lpinca  thanks for review/fix.

**Comment by aymen94** [comment]
@fhemberger I think it's time to close the pull request and do the merge.
What do you think? 😃

**Comment by lpinca** [comment]
@aymen94 as far as I can see none of my comments have been addressed. Forgot to push?

**Comment by aymen94** [comment]
> @aymen94 as far as I can see none of my comments have been addressed. Forgot to push?

oh yes, now I push.

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: come construtto

**Comment by aymen94** [comment]
@lpinca You can do the merge, so I can close PR 😄

File: locale/it/about/index.md
Comment (lpinca) [inline locale/it/about/index.md]: I think it's better to remove this. It seems a little out of context.

Comment (aymen94) [reply locale/it/about/index.md]: Is write on the English [page](https://nodejs.org/en/about/)

Comment (lpinca) [reply locale/it/about/index.md]: Ok then I'd would prefer to use the original translation "Node.js porta il modello ad eventi un po' oltre." which better reflect the corresponding English sentence "Node takes the event model a bit further" in my opinion.

**Review by lpinca — APPROVED** [review]

File: locale/it/about/index.md
Comment (aymen94) [reply locale/it/about/index.md]: You do the merge?

Comment (lpinca) [reply locale/it/about/index.md]: Let's keep it open until tomorrow just in case someone wants to chime in. We can then merge it. Thank you for your patience.

Comment (aymen94) [reply locale/it/about/index.md]: Ok, Thank you @lpinca.

**Comment by aymen94** [comment]
@fhemberger @lpinca 
I think it's complete.

**Comment by ghost** [comment]
@aymen94：Thanks for your contribution!

**Pull request merged**
